tsdb/series_partition.go

```diff
@@ -56,6 +58,7 @@ func NewSeriesPartition(id int, path string) *SeriesPartition {
 		CompactThreshold: DefaultSeriesPartitionCompactThreshold,
 		Logger:           zap.NewNop(),
 		seq:              uint64(id) + 1,
+		stats:            NewSeriesPartitionStats(),
```

Comment:
What happens on restart/re-open?  Are any stats accumulated prior to the restart lost?